stable/mongodb-replicaset/templates/mongodb-statefulset.yaml

```diff
@@ -1,4 +1,8 @@
+{{- if semverCompare "^1.9-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
No need for an `if` statement here. Let's move forward. Please update the prerequisites in the readme to 1.9+.